apps/storybook/src/Text.stories.tsx

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 import { Meta, Story } from '@storybook/react';
 import React from 'react';
-import { Text, TextProps } from '@itwin/itwinui-react/';
+import { Text, TextProps } from '@itwin/itwinui-react';
```

Comment:
So if the user imports with `/`, what error would they get?

Reply:
Something like this...
<img width="386" alt="Screenshot 2022-10-05 at 16 08 42" src="https://user-images.githubusercontent.com/81580355/194068038-cd67e3d2-08f9-43a0-86ec-4fa23f891504.png">

But i am sure, that the first time i got the error, it had `./` export was missing, not the `.`. Thats how I figured to global search import with ending slash...
Basically, user would always get export path is missing, if they are importing from path we have not defined in our package.json